test/SimplePDPService.t.sol

```diff
@@ -163,25 +163,29 @@ contract SimplePDPServiceFaultsTest is Test {
 
     function testFaultWithinOpenPeriod() public {
         pdpService.rootsAdded(proofSetId, 0, new PDPVerifier.RootData[](0), empty);
-        
```

Comment:
Where are these noisy edits coming from?  If we want to include them we should be running a linting check on CI and fix all of them at once.